client/blocks/jitm/index.jsx

```diff
@@ -29,6 +29,10 @@ function renderTemplate( template, props ) {
 		return <AsyncLoad { ...props } require="blocks/jitm/templates/sidebar-banner" />;
 	}
 
+	if ( template === 'upsell-nudge' ) {
+		return <AsyncLoad { ...props } require="blocks/jitm/templates/upsell-nudge" />;
```

Comment:
Instead of adding another template type I'd recommend adding an A/B test within the default template  https://github.com/Automattic/wp-calypso/blob/37eb81430afc89f5f7b114acf1083d83fd50475b/client/blocks/jitm/templates/default.jsx#L29

Since we plan on replacing banner usages with upsell-banner